<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="setupConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>建立 SimConnect 连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getCommanderData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>获取航班当前数据</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commanderData">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="killConnect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>断开 SimConnect 连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createOneAI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>创建一架 AI Plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteOneAI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>删除一架 AI Plane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setOneAI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>设置 AI 数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="createObjectId">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="createName">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="createWhatData">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>某参数？</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deleteObjectId">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setAlt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Alt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setLat">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Lat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setLon">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Lon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setHdg">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Hdg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setOnGround">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>OnGround</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setSpeed">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>AirSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ObjectId：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ObjectId：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Params？：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lat：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lon：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HDG：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OnGround：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Spd：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="connectStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
